lib/london_bike_hire_cli/basic_renderer.rb

```diff
@@ -1,38 +1,31 @@
+require 'erb'
+require_relative 'color_helper'
+
 module LondonBikeHireCli
   class BasicRenderer
+    include ColorHelper
+
     def initialize(output = STDOUT)
       @output = output
     end
 
-    def render(context)
-      output.print ">>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>\n"
-      output.print "Feed updated: #{context.first.display_feed_time}\n"
+    attr_reader :context
 
-      context.each do |station|
-        render_station(station)
-      end
-
-      output.print ">>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>\n"
-    end
-
-    def render_error(context)
-      output.print ">>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>\n"
-      output.print "Ooops, something went wrong with the query.\n"
-      output.print "The query #{context[:query].inspect}\n"
-      output.print "resulted in error: #{context[:error]}\n"
-      output.print "Try again?\n"
+    def render(template_name, context)
+      @context = context
+      output.print render_template(find_template(template_name))
     end
 
     private
 
     attr_reader :output
 
-    def render_station(station)
-      output.print ">>> Dock\n"
-      station.each_pair.each do |attr_name, attr_value|
-        output.print "#{attr_name.capitalize}: #{attr_value}\n"
-      end
-      output.print "Link to map: #{station.map_link}\n"
+    def render_template(template)
+      ERB.new(template).result(binding).gsub(/^\s+/, '')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.